### ISSUE TITLE
TestGrid Use Cases!

### ISSUE THREAD

**Issue by michelle192837** [GoogleCloudPlatform/testgrid#51]
Continuing from good old https://github.com/kubernetes/test-infra/issues/10409, this issue is open for anyone who wants to use TestGrid but is blocked on doing so while we're open-sourcing.

Add your requests/use cases here for how you'd like to use TestGrid! We'll use this to help determine how to prioritize work as we continue open-sourcing. ^^ (Feel free to look at the previous issue for inspiration, or file a separate issue with more details in addition to here!)

**Comment by RobertKielty** [comment]
1. As a TestGrid user I would like to be able to group jobs that I am interested in so that I can create a dashboard of those jobs displaying cards in a _responsive_ grid that present the following data about each job that I am interested in. 

Job Name
Last Run
 - Test Stats
 - Actual Duration  

Next Run 
 - Scheduled Time

Overall Trends Color 
Present the following as stacked horizontal bars of green, yellow, red 
 - Last 5 runs
 - Last 10 runs
 - Last 30 runs

2. As a Test Grid User when working with multiple jobs I would like to see the Title of the Job Page be the Job Name instead of the name of the Test Grid tab to which the Job belongs. When I am covering multiple jobs and have them open in multiple tabs at present the titles are all same which has a negative impact on the UX.

**Comment by RobertKielty** [comment]
There's an interesting amount overlap between TG Prow and Build Status the more I use these applications the better I can navigate results. The Build Status app (accessible from top right hand corner of a Prow Job result page) is _very_ useful and probably satisfies a lot of the use cases that I specified above.

**Comment by fejta** [comment]
I'm sure we can still improve the experience but you can follow https://github.com/GoogleCloudPlatform/testgrid/tree/master/cluster/prod and https://github.com/GoogleCloudPlatform/testgrid/blob/master/standalone.md to deploy and use your own public testgrid data.
